src/components/performance/PerformanceMonitor.tsx

```
@@ -0,0 +1,324 @@
+                variant="outline"
+                size="sm"
+                onClick={metrics.refresh}
+                disabled={metrics.isLoading}
+              >
+                <Refresh className={cn("h-4 w-4", metrics.isLoading && "animate-spin")} />
+              </Button>
+            </div>
+          </div>
+        </CardHeader>
+        <CardContent>
+          <Tabs value={selectedMetric} onValueChange={(value) => setSelectedMetric(value as any)}>
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)